test/models/bot/alegre_test.rb

```diff
@@ -23,6 +23,12 @@ def setup
           'confidence': 1.0
         }
       }.to_json)
+      Bot::Alegre.stubs(:request_api).returns({
+        'result' => {
+          'language' => 'en',
+          'confidence' => 1.0
+        }
+      })
```

Comment:
Please make sure to `unstub` at the end of the test (or at once in `def teardown`) because it doesn't happen automatically in this version of the library. Something like: `Bot::Alegre.unstub(:request_api)`.